feeds/feed.go

```diff
@@ -37,6 +40,15 @@ type Package struct {
 	SchemaVer   string    `json:"schema_ver"`
 }
 
+type FailedPackagePoll struct {
```

Comment:
We should name this more conventionally including `Error` in the name

Reply:
Agreed, changed to  `PackagePollError`